src/L.Control.Locate.js

```diff
@@ -283,6 +285,14 @@ You can find the project at: https://github.com/domoritz/leaflet-locatecontrol
                 link.role = 'button';
                 link.href = '#';
                 var icon = L.DomUtil.create(options.iconElementTag, options.icon, link);
+
+                if(options.strings.text !== undefined) {
+                    var text = L.DomUtil.create(options.textElementTag, 'leaflet-locate-text', link);
+                    text.textContent = '\xa0' + options.strings.text;
```

Comment:
Why do you need to add `\xa0`?

Reply:
> 
> 
> Why do you need to add `\xa0`?

To make a space between the icon and the text, otherwise they are stuck together and it's ugly. I had the choice to do it by css or like this, i choosen this method. But thinking about, maybe I should make the space by css on the <i></i> with a padding right, like this if the developer choose to use only the text without the icon there is no space for nothing. What do you think?

Reply:
Yep, let's do it in CSS.